Scripts/Modules/SerializedPreferencesModule.cs

```diff
@@ -94,8 +95,10 @@ internal string SerializePreferences()
 				}
 
 				if (item != null)
-					preferences.items.Add(item);
+					payloads[serializer.GetType()] = item;
```

Comment:
Hmm...Useful for us internally, but would be confusing to someone else. We would need some way to turn it on only for our builds. You could look for an internal define that only shows up when building from trunk?